lms/envs/common.py

```diff
@@ -2050,3 +2050,25 @@
 SEARCH_ENGINE = None
 # Use the LMS specific result processor
 SEARCH_RESULT_PROCESSOR = "lms.lib.courseware_search.lms_result_processor.LmsSearchResultProcessor"
+
+######################## Leaner profile configuration ###########################
+
+# Default visibility level for accounts without a specified value
+# The value is one of: 'global', 'edx_users', 'private'
+DEFAULT_PROFILE_VISIBILITY = 'edx_users'
```

Comment:
I don't think we should use "edx". And don't include "global" since we aren't implementing that yet.
I think "all_users", "private".